src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -454,6 +461,9 @@ enum AccessibilityIdentifiers {
 	}
 	
 	enum Statistics {
+		enum LocalStatistics {
+			
+		}
```

Comment:
Needed?

Reply:
no i moved it and forgot to delete this one, will push again